### ISSUE TITLE
Indien 421

### ISSUE THREAD

**Issue by manufrancis** [TST-Project/mss#4]
* About un-recorded leaf width and height (compulsory field): Are we satisfied with the value "0" for un-recorded width and height? Or shall we make this field optional? 
* Field "summary": display in bigger font.
* Field "extent": `<p>` added.
* Field "additions": add markup for titles in Tamil, but in very loose Roman transcription in the original. 
* Field "filiation": change display to "Other manuscript testimonies"?
* Field "bibliography": `<p>` added.

Add `<p>` among the tags automatically suggested when typing "<"
Add possibility to enlarge the writing boxes and seeing all encoded text.
Add @xml:lang="ta" to incipit and explicit. -- show me how to do it.

**Comment by chchch** [comment]
Updated Indien 421.
* deleted leaf measurements — it's optional now
* TEI doesn't allow `<p>` in `<extent>`... I've used `<foliation>` as in the other entries. Let's see what Axelle thinks!
* bibliography uses `<bibl>` instead of `<p>` for each bibliographical entry
* `@xml:lang` added automatically in the editor depending on the language of the msItem
* other items are done as suggested

Things still to think about
* how should we mark loosely-transliterated text? We could use `@xml:lang="ta-Latn"`, but that might conflict with our strict transliteration parts
